Utils/Dataflow/config/009.cfg.example

```diff
@@ -32,9 +32,17 @@ mode = SQUASH
 # initial timestamp (e.g. 01-05-2016 00:00:00)
 initial = %__009_init_offset__%
 # final timestamp (e.g. 10-05-2016 00:00:00)
+# If this parameter is not set, 'now' (current moment of time) will be used.
 final = %__009_final_offset__%
 # offset from current timestamp (e.g. 86400 <=> 86400s <=> 1d; 8h; 15m)
 step = %__009_step__%
+# delay (e.g. 86400 <=> 86400s <=> 1d; 8h; 15m)
+# If 'final' is not set, this parameter can be used to adjust 'now'.
+# For example, running the script at 10-05-2016 12:00:00 with
+# delay = 2h will set 'final' to 10-05-2016 10:00:00.
+# Has no effect if 'final' is set explicitly.
+# Cannot be negative.
```

Comment:
I was tending to type `delay: -10m`, indeed :)

Maybe it would be OK to allow negatives, but use `abs(delay)`? Or it will be confusing?

Reply:
Current version is simple and straightforward, while allowed negatives and `abs()` may complicate the situation a bit. I doubt that they'll be too confusing, but I'm not sure if we need it.
Another reason not to use the negative values is speed - as someone (here, in KI, or maybe even DKB) educated me at some point, strings such as `name-of-something` are considered to be better than `name_of_something` (which, for some reason, I preferred to use) because `-` is slightly faster to type than `_`. Same, I believe, goes for `10m` vs `-10m`.

Reply:
> strings such as `name-of-something` are considered to be better than `name_of_something` (which, for some reason, I preferred to use) because `-` is slightly faster/easier to type than `_`.
Same, I believe, goes for `10m` vs `-10m`.

I think that statement about `name-of-something` was related to the something that was supposed to be typed every now and then (filename, branch name, e-mail address, ...). Parameters in config files are not to write, but to read them fast ;)

But strictly positive delay is also OK, so let it be so.